cli/args/mod.rs

```diff
@@ -5,5 +5,442 @@ pub mod flags;
 
 mod flags_allow_net;
 
-pub use config_file::*;
+use std::collections::BTreeMap;
```

Comment:
Although a lot of this file is new, most of the code is just moved from other files.